src/main/java/com/danielagapov/spawn/Controllers/UserController.java

```diff
@@ -51,4 +52,11 @@ public UserDTO createUser(@RequestBody UserDTO newUser) {
     public UserDTO replaceUser(@RequestBody UserDTO newUser, @PathVariable UUID id) {
         return userService.replaceUser(newUser, id);
     }
+
+    // full path: /api/v1/user/friendReq
+    @PostMapping("friendReq")
```

Comment:
Going off of the resource @Daggerpov sent on Slack, `/user/{id}/friend-requests` would follow REST URI conventions closer.

Reply:
Good point, I'll revisit that site.